packages/validator-bonds-sdk/src/instructions/configureBondWithMint.ts

```diff
@@ -34,12 +33,8 @@ export async function configureBondWithMintInstruction({
   voteAccount?: PublicKey
   validatorIdentity?: PublicKey
   tokenAccount?: PublicKey
-  tokenAuthority?:
-    | PublicKey
-    | Keypair
-    | Signer
-    | WalletInterface
-    | WalletInterface // signer
+  tokenAuthority?: PublicKey | Keypair | Signer | WalletInterface
+  // signer
```

Comment:
```suggestion
  tokenAuthority?: PublicKey | Keypair | Signer | WalletInterface // signer
```
(probably auto-formatted to end up below the line it was meant for)